react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkRadioItem = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    radioAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    isDisabled,
+    onChangeFunc,
```

Comment:
This needs to be updated to onChange per our recent change we made to other inputs. Messaged @RVMendoza  on this!

Reply:
fixed